<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MigrateDialogBase</class>
 <widget class="QWizard" name="MigrateDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>507</width>
    <height>316</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Convert configuration from old version</string>
  </property>
  <widget class="QWizardPage" name="page1">
   <property name="title">
    <string>Select directories for convert</string>
   </property>
   <layout class="QVBoxLayout">
    <property name="margin">
     <number>11</number>
    </property>
    <item>
     <widget class="QLabel" name="TextLabel1">
      <property name="text">
       <string>Directories for convert:</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="Line" name="Line1">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="chkRemove">
      <property name="text">
       <string>&amp;Remove old files</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="TextLabel3">
      <property name="text">
       <string>Note!
If you select an option &quot; Remove old files &quot;, you can not use the old version</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="Spacer1">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWizardPage" name="page2">
   <property name="title">
    <string>Convert configuration</string>
   </property>
   <layout class="QVBoxLayout">
    <property name="margin">
     <number>11</number>
    </property>
    <item>
     <spacer name="Spacer2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="lblStatus">
      <property name="text">
       <string/>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QProgressBar" name="barCnv"/>
    </item>
    <item>
     <spacer name="Spacer3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Expanding</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <resources/>
 <connections/>
</ui>
